### PR TITLE
Preparing release version 5.3.1

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -18,6 +18,42 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+pytest 5.3.1 (2019-11-25)
+=========================
+
+No significant changes.
+
+
```
Comment (blueyed) [inline CHANGELOG.rst]: This is "duplicated".
Would be better to do this after https://github.com/pytest-dev/pytest/pull/6272 also, which improves the existing snippets already.

Comment (blueyed) [reply CHANGELOG.rst]: @nicoddemus 
Although #6272 is approved, I do not want to create a conflict here by merging it myself.
So please feel free to merge #6272 yourself if you think it is good either before, or after (but needs conflict resolution then - iff the improved changelogs make sense; I've merely done it to show what can be done there).

Comment (nicoddemus) [reply CHANGELOG.rst]: Sure, I will fix the conflicts after merging this one. 👍

Comment (nicoddemus) [reply CHANGELOG.rst]: (I will only be able to get to this much later though, so feel free to get at it if you want/can).

**Comment by hugovk** [comment]
Minor thing at https://github.com/pytest-dev/pytest/releases/tag/5.3.1, the newlines are a bit off:

---

image

---

Looks well justified in edit mode, but much of the markup isn't visible in the rendered version:

---

image

---

**Comment by nicoddemus** [comment]
@hugovk fixed, thanks!

**Comment by blueyed** [comment]
> the newlines are a bit off:

This applies to older releases also: https://github.com/pytest-dev/pytest/releases
towncrier config issue?

**Comment by nicoddemus** [comment]
> This applies to older releases also: pytest-dev/pytest/releases
towncrier config issue?

I think the problem is that technically Markdown ignores line-breaks inside paragraphs, but GitHub honours them.

**Pull request merged**